draft-scenarios/clash/gold_rush.tex

```diff
@@ -0,0 +1,145 @@
+% !TeX spellcheck = en_US
+\addscenariosection{1}{Solo/Clash Scenario}{Gold Rush}{\images/estates.png}
+
+\begin{multicols*}{2}
+
+    \textbf{Author:} farmaazon
+
+    \textbf{Source:} \href{https://discordapp.com/channels/740870068178649108/1344400556717768865/1344400556717768865}{Archon Studios Discord}
+
+    \textit{Your liege is in dire need of more gold for his long wars in far country. He laid on you a task of squeezing as much as you can from this area, and also find the legendary Grail.}
+
+    \subsection*{\MakeUppercase{Scenario Length}}
+
+    This Scenario is played over 13 Rounds.
+
+    \subsection*{\MakeUppercase{Player Setup}}
+
+    \textbf{Player Count:} 1, 2 or 4 (alliance variant)
+
+    \textbf{Starting Resources:} 15 \svg{gold}, 6 \svg{building_materials}, 1 \svg{valuables}
+
+    \textbf{Starting Income:} 10 \svg{gold}, 0 \svg{building_materials}, 0 \svg{valuables}
+
+    \textbf{Starting Units:}
+    \begin{itemize}
+        \item A Pack of \svgunit{bronze} Units with the hightst Recruitment cost
+        \item A Few \svgunit{silver} Units with the lowest Recruitment cost
+    \end{itemize}
+
+    \textbf{Town Buildings:} \svgunit{bronze} Dwelling
+
+    \textbf{Map Tile Pool:} Each player takes 3 Far (II--III) Tiles with at least one settlement.
```

Comment:
```suggestion
    \textbf{Map Tile Pool:} Each player takes 3 Far (II--III) Map Tiles with at least one Settlement.
```
Shouldn't it be exactly one settlement?

Reply:
> Shouldn't it be exactly one settlement?

This is how it was intended and playtested. I didn't see that additional settlements gave any significant advantage.